client/src/components/Layout/General/index.js

```diff
@@ -7,13 +7,26 @@ import Footer from "../../Footer";
 
 const General = ({ children, ...props }) => {
   const classes = useStyle(props);
-  const { header, sideBar, footer } = props;
+  const { header, sideBar, footer, navbar } = props;
+
+  const FooterCase = () => {
+    switch (true) {
+      case !!footer && !!navbar:
+        return <Footer navbar />;
+      case !!footer:
+        return <Footer />;
+      default:
+        return null;
+    }
+  };
```

Comment:
```suggestion
  const FooterCase = () => {
  if(!!footer && !!navbar) return  <Footer navbar />;
  if(!!footer) return <Footer navbar />;
    return null
  };
```
i have saw this pattern of using switch a lot in your latest PRs 
switch is not everything

Reply:
haha I just tried to avoid nested ternary, btw thanks for the suggestion,